tests/unit_tests/test_wave_06.py

```diff
@@ -110,16 +112,21 @@ def test_swap_best_by_category():
         their_priority="Decor"
     )
 
-    raise Exception("Complete this test according to comments below.")
+    #raise Exception("Complete this test according to comments below.")
     # *********************************************************************
     # ****** Complete Assert Portion of this test **********
     # *********************************************************************
     # Assertions should check:
     # - That the results is truthy
     # - That tai and jesse's inventories are the correct length
     # - That all the correct items are in tai and jesse's inventories, including the items which were swapped from one vendor to the other
+    assert result == True
+    assert len(tai.inventory) == 3
+    assert len(jesse.inventory) == 3
```

Comment:
I got to look at an older commit and I saw that you've updated it and cleaned it up! LMK if you have questions on this.

You've paired down these assert statements. For a fully tested feature, I'd expect just a few more details checked. From your previous commit, these are the assert lines I would pull out and keep:

```python
    assert result == True
    assert len(tai.inventory) == 3
    assert len(jesse.inventory) == 3
    assert item_a in tai.inventory 
    assert item_b in tai.inventory 
    assert item_d not in tai.inventory 
    assert item_e not in tai.inventory 
    assert item_d in jesse.inventory 
    assert item_e in jesse.inventory 
    assert item_a not in jesse.inventory 
    assert item_b not in jesse.inventory 
```


Since I was grading your previous commit of this project, I want to copy/paste in my comments just generically here:

```python
    # When you originally had these lines:
    assert result in tai.inventory
    assert result in jesse.inventory
```

Earlier in this test, you check if `result` is equal to `True`. This line checks if the value `True` is anywhere inside the list `tai.inventory`. For example, the test error was:

```
assert True in [<swap_meet.electronics.Electronics object at 0x101e9a950>, <swap_meet.decor.Decor object at 0x101e9a7d0>, <swap_meet.clothing.Clothing object at 0x101e99690>]
```

And when you originally had the lines:

```python
    assert len(tai.inventory) == 3
```

My comments are:
This test fails because it says `AttributeError: 'Vendor' object has no attribute 'intventory'`. That's because in this line, it's looking for `intventory`... when `Vendor` instances have an `inventory` :') ...
Fixing this typo helps the test get passed this error.